src/device.h

```diff
@@ -12,6 +12,10 @@
 
 #ifndef MG_DEVICE
 #define MG_DEVICE MG_DEVICE_NONE
+#elif MG_DEVICE == MG_DEVICE_STM32H7
+#define MG_BOOTLOADER_SECTION __attribute__((section(".bootloader")))
```

Comment:
```c
#define MG_IRAM ...
```

Reply:
That won't work on non gcc-based compilers... see my comment regarding reinventing CMSIS